test/gc/sigmaskgc.d

```diff
@@ -0,0 +1,42 @@
+
+// https://issues.dlang.org/show_bug.cgi?id=20256
+
+extern(C) __gshared string[] rt_options = [ "gcopt=parallel:1" ];
+
+void main()
+{
+    version (Posix)
+    {
+        import core.sys.posix.signal;
+        import core.sys.posix.unistd;
+        import core.thread;
+        import core.memory;
+
+        sigset_t m;
+        sigemptyset(&m);
+        sigaddset(&m, SIGHUP);
+
+        auto x = new int[](10000);
+        for (int i=0; i<10000; i++)
```

Comment:
code style nit: `for (int i = 0; i < 10000; i++)`

Reply:
Bonus for `foreach (i; 0 .. 10000)`